spec/features/node_pages_spec.rb

```diff
@@ -3,7 +3,10 @@
 describe "node pages" do
   subject { page }
 
-  before { login_to_project_as_user }
+  before do
+    ActiveJob::Base.queue_adapter = :test
```

Comment:
Why do this here? Couldn't it just go in `rails_helper`?

Reply:
@georgemillo you are right, changed.